src/components/Text/Text.tsx

```diff
@@ -32,9 +33,11 @@ const Text: VibeComponent<TextProps, HTMLElement> & {
       ...typographyProps
     },
     ref
-  ) => {
+    ) => {
+    const { ignoreHeightOverflow } = useContext(TextContext);
```

Comment:
That's not right. You should call `useContext` in the `Typography` component, then pass this value into `useTooltipProps` which should pass it further to `useIsOverflowing` hook